tests/ui_tests/app/page/common/table/locator.py

```diff
@@ -24,6 +24,7 @@ class CommonTable:
 
     header = Locator(By.XPATH, "//mat-header-cell/div", "Table header")
     row = Locator(By.XPATH, "//mat-row", "Table row")
+    visible_row = Locator(By.XPATH, "//mat-row[@adwphover]", "Visible table row")
```

Comment:
visible_row = Locator(By.CSS_SELECTOR, "mat-row[adwphover]", "Visible table row")

Reply:
Thanks, changed here and below